octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java

```diff
@@ -134,6 +137,9 @@ public void expansionStateChanged(ExpansionEvent expansionEvent) {
         PrefereceChangeHandler prefereceChangeHandler = () -> drawEntityFields(entityModel);
         PluginPreferenceStorage.addPrefenceChangeHandler(PreferenceConstants.SHOWN_ENTITY_FIELDS, prefereceChangeHandler);
         addDisposeListener(e -> PluginPreferenceStorage.removePrefenceChangeHandler(PreferenceConstants.SHOWN_ENTITY_FIELDS, prefereceChangeHandler));
+        
+        //initialize fieldEditors
+        fieldEditors = new ArrayList<FieldEditor>();
```

Comment:
Is there a reason why you didn't do this when declaring the variable? I see that you're not waiting for any other value to be computed before initializing this param. It's a better code practice to try to initialize where you're declaring, or closer to where you're declaring the variable.